crates/store/re_types/definitions/rerun/archetypes/ellipsoids.fbs

```diff
@@ -0,0 +1,63 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+include "cpp/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/components.fbs";
+
+namespace rerun.archetypes;
+
+// ---
+
+/// 3D ellipsoids with half-extents and optional center, rotations, rotations, colors etc.
+///
+/// This archetype is for ellipsoids or spheres whose size is a key part of the data
+/// (e.g. a bounding sphere).
+/// For points whose radii are for the sake of visualization, use `Points3D` instead.
+///
+/// Currently, ellipsoids are always rendered as wireframes.
+/// Opaque and transparent rendering will be supported later.
+///
+/// \example archetypes/ellipsoid_batch !api title="Batch of ellipsoids"
```

Comment:
I cannot find this example. Please add it and update the PR description with instructions on how to test the PR!

Reply:
Doesn't this refer to `docs/snippets/all/archetypes/ellipsoid_batch.{rs,py,cpp}`? That was my intent.

Reply:
Huh, I didn't find it, but I can see it's there now. Weird